### PR TITLE
# Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by Marcraphael12**
In this PR we are initializing our project with React components

### Project requirements
- Initialise React app.
- The *building blocks* of your app should be set as reusable components. Create a directory for your components.
- Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two `<Route>`s and `<Link>`s for the app's navigation:
    - Books - the default view:
        - Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet).
            - Create a component called Book for displaying a single book (receive a title and an author as a prop) and reuse it in a component that displays a list of books.
        - Should display a form for adding a book (no functionality yet).
            - Create a separate component for this form (with inputs for a title and an author [**IMPORTANT** in the design you can see an input for a category - please replace it with an author.]).
    - Categories:
        - Should display a button "Check status" only.
- Styling is not required at this point.

### CODE REVIEW

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi @Marcraphael12 🙋‍♀️,  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
 
You have submitted a project, that has a README that is not documented. Please update all the sections on the README and make them relevant to the project. Please do not remove any sections except for the FAQ section because it is optional.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by mirouhml** [comment]
Hi @Marcraphael12, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a complete README, please know that all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present, the only optional section is FAQ, also, make sure to add the section Future Features, including 1-3 potential features to be added.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/categories/Categories.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function categories() {
+  return (
+    <div>
+      <h2>Under construction</h2>
+    </div>
```
Comment (rotimiazeez) [inline src/components/categories/Categories.js]: - Please note that when you click on the categories, it should display only a button with text "Check status" 👍🏽

File: src/components/books/AddBook.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <option value="Science Fiction">Science Fiction</option>
+        <option value="Economy">Economy</option>
+      </select>
+    </label>
+
+    <button type="submit">Add Book</button>
+  </form>
+);
+
+export default AddBookForm;
```
Comment (rotimiazeez) [inline src/components/books/AddBook.jsx]: Screenshot_20230121-175423_Chrome
- Kindly make sure a form for adding a new book is rendered on your web app, only as a placeholder for now, but it should be displayed 😉

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Marcraphael12 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
### You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

Please feel free to use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: src/components/categories/Categories.js (hunk 1 above)
Comment (Marcraphael12) [reply src/components/categories/Categories.js]: done thanks👍🏽

File: src/components/books/AddBook.jsx (hunk 2 above)
Comment (Marcraphael12) [reply src/components/books/AddBook.jsx]: done thanks👍🏽... for me it's showing... anyway, I converted it into a function

**Review by BregornOriginal — APPROVED** [review]
## Hi @Marcraphael12 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 

## Congratulations! 👏🏼 👏🏼 👏🏼 

approve-applause

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good pull request description


## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**